webdriver/tests/bidi/network/continue_request/invalid.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import webdriver.bidi.error as error
+
+pytestmark = pytest.mark.asyncio
```

Comment:
I assume we should have that after all the imports?

Reply:
Sure, updated.